rust/src/dcerpc/log.rs

```diff
@@ -22,7 +22,7 @@ use crate::jsonbuilder::{JsonBuilder, JsonError};
 fn log_dcerpc_header(
     jsb: &mut JsonBuilder, state: &DCERPCState, tx: &DCERPCTransaction,
 ) -> Result<(), JsonError> {
-    if tx.req_done == true {
+    if tx.req_done == true && tx.req_lost == false {
```

Comment:
Is it not worth to log the request alone ?

Reply:
I'm not sure I understand what you mean here. This check was added because in case the request was lost, we won't have the params that we try to access for logging.
Could you please explain what you think should have been done here?

Reply:
Indeed, you are right :-)